### PR TITLE
Automate copyright current year

### PR DESCRIPTION
**Pull request by elecpower**
The current year is manually maintained and has not been updated for 2.3.8 so here is my attempt to automate.
I have compiled Companion in Ubuntu and the About dialog is fixed but I do not have a Windows dev environment to test the NSIS installer (so help here would be appreciated, thanks in advance)

### CODE REVIEW

File: companion/targets/windows/license.txt.in [hunk 1]
```diff
@@ -1,6 +1,6 @@
-Copyright � 2011-2019 OpenTX team
+Copyright � 2011-@WIN_COMPILE_YEAR@ OpenTX Team
```
Comment (bsongis) [inline companion/targets/windows/license.txt.in]: I am afraid you have broken the file encoding?

**Comment by elecpower** [comment]
@bsongis The attempted fix did not work. I think it is related to my Git repo file encoding.  The file maintains the encoding in the editor. Its in the git compare where the issue shows up.
I have my editor (Code::Blocks) default and Git repo (SmartGit) set to UTF-8. Where am I going wrong?

**Comment by elecpower** [comment]
@bsongis Changing SmartGit to Windows-1252 seems to fix the diff in my repo but latest commits are not resulting in the desired result so I'll close the PR and create a new one from scratch.

**Comment by bsongis** [comment]
thanks! we have made such a mistake some years ago, and there was a strange char displayed on the end user program ...

**Pull request closed without merging**